src/foam/parsers/HTMLLexer.js

```diff
@@ -105,30 +68,21 @@ foam.CLASS({
           htmlPart: alt(
               sym('cdata'),
               sym('comment'),
-              sym('text'),
-
-              // "embed" is specific tag types; must come before "openTag".
-              sym('embed'),
-              sym('maybeEmbed'),
-
               sym('closeTag'),
-              sym('openTag')),
+              sym('openTag'),
+              sym('text')),
 
           openTag: seq(
               '<',
-              sym('whitespace'),
```

Comment:
The HTML Standard does not allow for whitespaces to exist between the opening angled bracket and the identifier